skale/dataclasses/tx_res.py

```diff
@@ -22,6 +22,10 @@ class TransactionFailedError(Exception):
     pass
 
 
+class DryRunFailedError(Exception):
```

Comment:
I think it should has same base class as transaction failed error

Reply:
`TransactionFailedError` has the same base class:
```
class TransactionFailedError(Exception):
    pass

class DryRunFailedError(Exception):
    pass
```

Reply:
I mean base class other than Exception. If we for example need to catch both to check if transaction failed in general.